nests/ubuntu-20.04-mspdev/development.txt

```diff
@@ -0,0 +1,23 @@
+attrs
```

Comment:
Do we need to have this here? This list of development dependencies is pretty fluid and it'll get out of date quickly. It'd be better to pip install the dev dependencies from the msprime/tskit repo directly I'd say.

Reply:
No need to have it here. I'm exploring and will commit a less hacky approach. Still a bit hacky, but less hacky.
There is also duplication between the list of required deb package names in the Dockerfile and the list of deb package names in the installation instructions. Might be worth having a single script file which contains the platform specific commands to run, have that file near msprime/requirements/*.txt, and then have both doc generation and this container building script read the same file to run the platform specific required system package installing lines.

Reply:
Commit in PR branch now, with this duplicated file deleted.
The Dockerfile is in theory less reproducible now. But the reproducibility of a container images from it's Docker file directory IMO is on a spectrum. I think having ~/src/msprime and a comment that it's a local git clone of the msprime github repository URL is reproducible enough for it's purpose.
That said, this might mean this doesn't work with `docker` even though `buildah` (podman) is fine with this. I stopped using docker when podman came out because docker, at least early versions of it, had a maniacal aversion to allowing functionality which would prevent reproducibility.